youtube_dl/extractor/francetv.py

```diff
@@ -360,3 +360,73 @@ def _real_extract(self, url):
             webpage, 'video id').split('@')
 
         return self._extract_video(video_id, catalogue)
+
+
+class JeunesseBaseIE(FranceTVBaseInfoExtractor):
+    def _real_extract(self, url):
+        id = self._match_id(url)
```

Comment:
Don't shadow built-in names.

Reply:
@dstftw : Thx again for your review, what do you mean ?  Do you want me to copy/paste the same `_real_extract` function in `ZouzousIE` and `LudoIE` ?

Reply:
ah... maybe you talked about `id`.